src/Components/Publishing/Fixtures/Components.ts

```diff
@@ -351,6 +351,16 @@ export const ImageSetMiniSansTitle: ImageSetPreviewProps["section"] = {
   images: Images,
 }
 
+export const Media = [
+  {
+    title: "Trevor Paglan",
+    url: "http://files.artsy.net/videos/scenic_mono_3.mp4",
+    duration: 560948,
```

Comment:
Here is another instance of the duration -- was using strings here but we can update articleCard to read this.